tools/apiview/parsers/swagger-api-parser/SwaggerApiParser/SwaggerApiView/SwaggerApiViewOperation.cs

```diff
@@ -102,32 +100,23 @@ public CodeFileToken[] TokenSerialize(SerializeContext context)
         // new line for `Parameters` section.
         ret.Add(TokenSerializer.NewLine());
 
-        // ret.Add(TokenSerializer.Intent(context.intent));
```

Comment:
Can you please remove these commented code?

Reply:
Removed